conf/drupal/config/core.entity_view_display.media.document.linked_desc.yml

```diff
@@ -28,38 +28,38 @@ content:
   calculated_file_size:
     settings: {  }
     third_party_settings: {  }
-    weight: 3
+    weight: 2
```

Comment:
Why do we need changes in config?

Reply:
replaced with the latest version of the configs.  same to the other below.